ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/ConversionsSpec.scala

```diff
@@ -188,39 +221,76 @@ class ConversionsSpec extends AnyWordSpec with Matchers with OptionValues {
         "handle with expected status codes" in {
           forAll(
             Table(
-              ("rejection builder", "code"),
+              ("rejection builder", "code", "expectedAdditionalDetails"),
               (
                 (builder: DamlTransactionRejectionEntry.Builder) =>
```

Comment:
You can get rid of this boilerplate by declaring an explicit type for the `Table`, i.e., `Table[DamlTransactionRejectionEntry.Builder => DamlTransactionRejectionEntry.Builder, Status.Code]`. Then, the builders can be constructed simply as e.g. `_.setInconsistent(...)`.

Reply:
Great advice, thanks!